source/layouts/layout.html.haml

```diff
@@ -2,7 +2,7 @@
   <!--[if IE]><script src='http://html5shiv.googlecode.com/svn/trunk/html5.js'></script><![endif]-->
   %meta{charset:'utf-8'}
   <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
-  %meta{content:'width=device-width,initial-scale:1.0,maximum-scale:1.0', name:'viewport'}
+  %meta{content:'width=device-width', name:'viewport'}
```

Comment:
I'm not really sure, which is the other reason I thought to remove it. I
must have copy-pasta from someone else. I'm going to keep it out for now -
the zooming thing seems annoying.



-- 
@adarshp <http://www.twitter.com/adarshp>
313.454.1515 (mobile)
See when I am free: http://bit.ly/adarshcal

On Tue, Nov 10, 2015 at 9:45 AM, Gabriel Berke-Williams <
notifications@github.com> wrote:

> In source/layouts/layout.html.haml
> <https://github.com/adarsh/blog/pull/24#discussion_r44438832>:
>
> > @@ -2,7 +2,7 @@
> >    <!--[if IE]><script src='http://html5shiv.googlecode.com/svn/trunk/html5.js'></script><![endif]-->
> >    %meta{charset:'utf-8'}
> >    <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
> > -  %meta{content:'width=device-width,initial-scale:1.0,maximum-scale:1.0', name:'viewport'}
> > +  %meta{content:'width=device-width', name:'viewport'}
>
> Why set maximum-scale=1, though? I think that prevents people from
> zooming at all, which is pretty annoying.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adarsh/blog/pull/24/files#r44438832>.
>